src/rabbit_control_main.erl

```diff
@@ -20,8 +20,8 @@
 -include("rabbit_misc.hrl").
 
 -export([start/0, stop/0, parse_arguments/2, action/5, action/6,
-         sync_queue/1, cancel_sync_queue/1, become/1,
-         purge_queue/1]).
+    sync_queue/1, cancel_sync_queue/1, become/1,
```

Comment:
I suggest reverting this indentation change to make multi-line lists in line with how they are formatted in the rest of the file.